examples/ensemble/plot_adaboost_multiclass.py

```diff
@@ -1,52 +1,56 @@
-r"""
+"""
 =====================================
 Multi-class AdaBoosted Decision Trees
 =====================================
 
-This example reproduces Figure 1 of Zhu et al [1]_ and shows how boosting can
-improve prediction accuracy on a multi-class problem. The classification
-dataset is constructed by taking a ten-dimensional standard normal distribution
-and defining three classes separated by nested concentric ten-dimensional
-spheres such that roughly equal numbers of samples are in each class (quantiles
-of the :math:`\chi^2` distribution).
-
-The performance of the SAMME and SAMME.R [1]_ algorithms are compared. SAMME.R
-uses the probability estimates to update the additive model, while SAMME  uses
-the classifications only. As the example illustrates, the SAMME.R algorithm
-typically converges faster than SAMME, achieving a lower test error with fewer
-boosting iterations. The error of each algorithm on the test set after each
-boosting iteration is shown on the left, the classification error on the test
-set of each tree is shown in the middle, and the boost weight of each tree is
-shown on the right. All trees have a weight of one in the SAMME.R algorithm and
-therefore are not shown.
-
-.. [1] J. Zhu, H. Zou, S. Rosset, T. Hastie, "Multi-class AdaBoost", 2009.
+This example shows how boosting can improve prediction accuracy on a multi-class
+problem by reproducing Figure 1 of [1].
 
+- [1] J. Zhu, H. Zou, S. Rosset, T. Hastie, "Multi-class AdaBoost", 2009.
```

Comment:
The previous `..` is the way sphinx will create an hyperlink such that `[1]` in the text will be redirected to this link.
So we need to keep it. For consistency with other improved examples I proposed the following changes:


```suggestion
.. topic:: References:
    .. [1] J. Zhu, H. Zou, S. Rosset, T. Hastie,
       `"Multi-class AdaBoost"
       <https://hastie.su.domains/Papers/samme.pdf>`_,
       2009.
```

I also add an external hyperlink to the original article. It is easier to get the reference to the Fig. 1.